src/dmd/clone.d

```diff
@@ -141,20 +145,32 @@ extern (C++) FuncDeclaration hasIdentityOpAssign(AggregateDeclaration ad, Scope*
 private bool needOpAssign(StructDeclaration sd)
 {
     //printf("StructDeclaration::needOpAssign() %s\n", sd.toChars());
-    if (sd.isUnionDeclaration())
+
+    static bool need()
```

Comment:
Naming conventions, as far as I'm aware, would prefer `isNeeded()` and `isNotNeeded`.

Reply:
Do we really need both?